Gemfile

```diff
@@ -7,27 +7,43 @@ source "https://rubygems.org"
 #
 # This will help ensure the proper Jekyll version is running.
 # Happy Jekylling!
-gem "jekyll", "~> 4.3.0"
-gem "webrick", "~> 1.7"
-# If you want to use GitHub Pages, remove the "gem "jekyll"" above and
-# uncomment the line below. To upgrade, run `bundle update github-pages`.
-#gem "github-pages", group: :jekyll_plugins
+gem "jekyll", "~> 4.3.2"
+gem "jekyll-theme-legumeinfo", path: "./_themes/jekyll-theme-legumeinfo"
+gem "logger", "~> 1.7.0"
+gem "csv", "~> 3.3.4"
+gem "base64", "~> 0.2.0"
+gem "bigdecimal", "~> 3.1.8"
+gem "bundler", "~> 2.6.9"
```

Comment:
XCode 16 has bundler 1.17.2, resulting in an error w/ `make install`. Suggest removing version:

```
gem "bundler"
```